### ISSUE TITLE
radio-groupname-case.html still fails one caseless comparison test

### ISSUE THREAD

**Issue by jdm** [servo/servo#7818]
https://github.com/servo/servo/pull/7777/files#diff-ed2c588bed7bbd35cf3ebc3cc09ded10R3

This points at an incomplete implementation in https://github.com/SimonSapin/rust-caseless, but we should figure out precisely what the failure case is in the testcase after #7777 merges.

cc @KiChjang in case you're interested in tackling this :)

**Comment by KiChjang** [comment]
I think I'll dive into this one once I get some of my other stuff done. In the meantime, if anyone is interested in this issue, go ahead ;)

**Comment by psdh** [comment]
This seems to fail on https://github.com/w3c/web-platform-tests/blob/master/html/semantics/forms/the-input-element/radio-groupname-case.html#L17 (on `&#x212a;` to be specific). But at the same time, line number 34 doesn't fail.

**Comment by KiChjang** [comment]
For reference, `&#x212a;` is the Kelvin sign used for measuring temperature.

**Comment by psdh** [comment]
@KiChjang Yep it is, the same kelvin sign matches correctly to 'k' (006b) in line 34. Not sure why that is happening

**Comment by psdh** [comment]
As SimonSapin pointed out, [html spec](https://html.spec.whatwg.org/multipage/infrastructure.html#compatibility-caseless) says "no language-specific tailorings", and [Unicode's CaseFolding.txt](http://www.unicode.org/Public/UNIDATA/CaseFolding.txt) says about T "For non-Turkic languages, this mapping is normally not used."

**Comment by SimonSapin** [comment]
I think this is a test bug: https://github.com/w3c/web-platform-tests/issues/2579

**Comment by annevk** [comment]
FYI: HTML changed in https://github.com/whatwg/html/pull/1941.

**Comment by nox** [comment]
This seems to still be relevant.

**Comment by simonwuelker** [comment]
We now pass 7/7 subtests (https://wpt.fyi/results/html/semantics/forms/the-input-element/radio-groupname-case.html?label=master&label=experimental&product=servo)
